.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: Release Actions
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  eb:
+    name: experimental-binaries
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      actions: read
+    steps:
+      - name: Create Dockerfile
+        run: |
+          cat <<EOF > Dockerfile
+          FROM scratch
+          LABEL org.opencontainers.image.description="A collection of experimental Leap binary packages"
+          COPY *.deb /
+          EOF
+      - name: Get ubuntu20 leap-dev.deb
+        uses: AntelopeIO/asset-artifact-download-action@v3
+        with:
+          owner: ${{github.repository_owner}}
+          repo: ${{github.event.repository.name}}
+          file: leap-dev.deb
+          target: ${{github.sha}}
+          artifact-name: leap-dev-ubuntu20-amd64
+          wait-for-exact-target-workflow: true
```

Comment:
Compared to #1795, `fail-on-missing-target: false` has been removed. I added that so the workflow would work on 3.x or 4.x or 5.x, but that's not needed since the workflow on the tagged ref is used.